pkg/controller/apischeme/apischeme_controller.go

```diff
@@ -257,12 +260,29 @@ func (r *ReconcileAPIScheme) Reconcile(ctx context.Context, request reconcile.Re
 		// no problems
 		r.SetAPISchemeStatus(instance, "Success", "Admin API Endpoint created", cloudingressv1alpha1.ConditionReady)
 		r.SetAPISchemeStatusMetric(instance)
-		return reconcile.Result{RequeueAfter: 60 * time.Second}, nil
+		return reconcile.Result{RequeueAfter: longwait * time.Second}, nil
 	case *cioerrors.DnsUpdateError:
 		// couldn't update DNS
 		r.SetAPISchemeStatus(instance, "Couldn't reconcile", "Couldn't ensure the admin API endpoint: "+err.Error(), cloudingressv1alpha1.ConditionError)
 		r.SetAPISchemeStatusMetric(instance)
 		return reconcile.Result{}, err
+	case *cioerrors.ForwardingRuleNotFoundError:
+		// This error handles the missing/deleted forwarding rule/LB in cloud provider
+		r.SetAPISchemeStatus(instance, "Couldn't reconcile", "Forwarding rule was deleted on cloud provider", cloudingressv1alpha1.ConditionError)
+
+		// To recover from this case we will need to delete the lb service.
+		// It will be recreated  at the next reconcile.
+		reqLogger.Info(fmt.Sprintf("Forwarding rule was deleted on cloud provider, deleting service %s/service/%s to force recreation", found.GetNamespace(), found.GetName()))
```

Comment:
This error only affects GCP, so I think its safe to say GCP rather than cloud proivder